docs/modules/configuration/partials/nav.adoc

```diff
@@ -13,4 +13,5 @@
 *** xref:configuration:dynamic-config-programmatic-api.adoc[Programmatic API]
 *** xref:configuration:dynamic-config-update-and-reload.adoc[Update and Reload with REST]
 *** xref:configuration:dynamic-config-persistence.adoc[Configuration Options]
+** xref:cluster-performance:thread-per-core-tpc.adoc[]
```

Comment:
As discussed - lets move this down towards Tiered Storage.
Lets put TPC and Tiered Storage, below the Replicating data across clusters subsection.

Otherwise, looks ok to me.